cycledash/genotypes.py

```diff
@@ -383,8 +383,31 @@ def _header_spec(vcf_header_text, extant_cols):
         path=['sample_name']) # This path is not the default super -> sub column
 
     # Add Cycledash-derived columns
-    column_name = 'annotations:gene_names'
-    if column_name in extant_cols:
+    _add_extant_column_to_spec(extant_cols, 'annotations:gene_names', res, 'String', 1,
```

Comment:
I can't tell if it's just some funky GitHub formatting, but some of these lines around here may be longer than 80 charts; mind chopping them down to size?